Sources/Starknet/Providers/StarknetProvider/JsonRpcMethod.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-enum JsonRpcMethod: String, Encodable {
+public enum JsonRpcMethod: String, Encodable {
```

Comment:
note: Adding public init to `StarknetRequest` enforces to make `JsonRpcMethod` public.

Reply:
Having looked at the code more closely, I agree that it's not necessary for `StarknetRequest.init` to be public as we will always use the "builder" methods to create a request 👍

Reply:
Removed 👍